### PR TITLE
Fixed broken TypeScript type definitions

### LINKED ISSUE #229: TypeScript types are broken after last updates
Compiling with TypeScript is actually not possible
```
Using tsc v2.4.2
node_modules/react-player/index.d.ts(32,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(32,5): error TS1131: Property or signature expected.
node_modules/react-player/index.d.ts(39,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(39,5): error TS1131: Property or signature expected.
node_modules/react-player/index.d.ts(46,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(46,5): error TS1131: Property or signature expected.
node_modules/react-player/index.d.ts(49,4): error TS1005: ';' expected.
node_modules/react-player/index.d.ts(49,5): error TS1131: Property or signature expected.
```
PR follows

### PR DESCRIPTION
**Pull request by k7sleeper**
Fix #229

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,7 +1,19 @@
 import * as React from 'react';
 
+export interface SourceSpec {
```
Comment (CookPete) [inline index.d.ts]: Is there any point `export` ing `SourceSpec` and `TrackSpec`? They are only used in this file.

File: index.d.ts [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+  src: string;
+  type: string;
+}
+
+export interface TrackSpec {
```
Comment (CookPete) [inline index.d.ts]: I think the names `SourceProps` and `TrackProps` make more sense?

File: index.d.ts (hunk 1 above)
Comment (k7sleeper) [reply index.d.ts]: > They are only used in this file.
Sure?
If you want to prepare an object of type `SourceSpec` somewhere in your source you'll be happy to be able to import that type.

File: index.d.ts (hunk 2 above)
Comment (k7sleeper) [reply index.d.ts]: +1

**Pull request merged**